content/zh-cn/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -255,6 +255,31 @@ scores from all plugins according to the configured plugin weights.
 在[标准化评分](#normalize-scoring)阶段之后，
 调度器将根据配置的插件权重合并所有插件的节点分数。
 
+<!--
+#### Capacity scoring {#scoring-capacity}
+
+{{< feature-state feature_gate_name="StorageCapacityScoring" >}}
+
+The feature gate `VolumeCapacityPriority` was used in v1.32 to support storage that are
+statically provisioned. Starting from v1.33, the new feature gate `StorageCapacityScoring`
+replaces the old `VolumeCapacityPriority` gate with added support to dynamically provisioned storage.
+When `StorageCapacityScoring` is enabled, the VolumeBinding plugin in the kube-scheduler is extended
+to score Nodes based on the storage capacity on each of them.
+This feature is applicable to CSI volumes that supported [Storage Capacity](/docs/concepts/storage/storage-capacity/),
+including local storage backed by a CSI driver.
+-->
+#### 容量打分 {#scoring-capacity}
+
+{{< feature-state feature_gate_name="StorageCapacityScoring" >}}
+
+在 v1.32 版本中，特性门控开关 `VolumeCapacityPriority` 被用于支持静态预配置的存储。
```

Comment:
```suggestion
在 v1.32 中，特性门控 `VolumeCapacityPriority` 被用于支持静态制备的存储。
```
Suggest to follow the glossary:
https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#glossary